tests/scripts/build-release-downstream.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex
+
+# Load dot env file if available
+if [ -f .env ]; then
+    # shellcheck disable=SC2046
+    export $(grep -v '^#' .env | xargs -d '\n')
+fi
+
+MAKE='make --debug=v --output-sync'
+$MAKE IMAGES="ceph" build
+build_Image=$(docker images | grep build | awk '{print $1; exit}')
+git_hash=$(git rev-parse --short "${GITHUB_SHA}")
+tag_Image=quay.io/paarora/rook-ceph-operator:v${BRANCH_NAME}$git_hash
+docker tag "$build_Image" "$tag_Image"
+docker push "$tag_Image"
```

Comment:
```suggestion
docker push "$tag_Image"
```

we also need to push the bundle image

Reply:
can you add the cmds ho we can create the bundle, or maybe a separate PR for that?

Reply:
`make bundle` is the cmd to create the bundle image, then we. an simply push the generated image.

Reply:
> can you add the cmds ho we can create the bundle, or maybe a separate PR for that?

let's make that change in this PR as that will be related change only